ooniprobe/Model/Database/Measurement.mm

```diff
@@ -50,6 +50,15 @@ -(NSString*)getLogFile{
     return [self.result_id getLogFile:self.test_name];
 }
 
+-(NSString*)getLocalizedStartTime{
+    //TODO we might need a copy for this
+    if (self.start_time == nil)
+        return NSLocalizedString(@"TestResults.UnknownASN", nil);
```

Comment:
Should this not be a different message string?

Reply:
No, as I decided with @agrabeli, as this is a temp fix we use this string to don't wait for translations. This fix is not present in Android and will be removed in iOS sometime in the future

Reply:
Ok I see. Maybe `TestResults.NotAvailable` is more appropriate though.